tests/sentry/integrations/opsgenie/test_integration.py

```diff
@@ -1,14 +1,24 @@
 import pytest
 import responses
+from rest_framework.serializers import ValidationError
 
 from sentry.integrations.opsgenie.integration import OpsgenieIntegrationProvider
 from sentry.models.integrations.integration import Integration
 from sentry.models.integrations.organization_integration import OrganizationIntegration
 from sentry.shared_integrations.exceptions import IntegrationError
 from sentry.testutils import IntegrationTestCase
+from sentry.testutils.silo import control_silo_test
 from sentry.utils import json
 
+EXTERNAL_ID = "test-app"
+METADATA = {
+    "api_key": "1234-ABCD",
+    "base_url": "https://api.opsgenie.com/",
+    "domain_name": "test-app.app.opsgenie.com",
+}
 
+
+@control_silo_test
```

Comment:
```suggestion
@control_silo_test(stable=True)
```

The stable parameter gives better checks on cross silo operations.